src/lib/route-data/route-data.interface.ts

```diff
@@ -0,0 +1,5 @@
+import { LabelFunction } from './route-data-function-types';
+
+export interface RouteDataInterface {
+    label: string | LabelFunction;
```

Comment:
optional oder pflicht? in terra ist es optional

Reply:
ja aber überall wird nen label angegeben und ohne gehen ja auch die breadcrumbs nicht

Reply:
müssen halt schauen ob das dann kompatibel ist mit dem `TerraRouteDataInterface`

Reply:
ja lass erst mal die Klasse erstellen, können wir ja dann ggf noch ändern